### DISCUSSION TITLE
I want to immerse myself in the blockchain community

### DISCUSSION THREAD

**Discussion by gomandev** [Cyfrin/foundry-full-course-cu#2445] [General]
Hi Guys,

I just finished Patrick's lesson 1 and want to be part of what is firing right now in the blockchain community. Can anyone recommend a group discussion channel on Discord, LinkedIn, or any platform?

Cheers. 🙏🏾

**Comment by EngrPips** [comment]
Are you on the official discord of Cyfrin https://discord.gg/t8qVHJCb
